src/app/app.component.ts

```diff
@@ -15,31 +16,35 @@ export class ClickerApp {
   private platform: Platform;
   private splash: SplashScreen;
   private status: StatusBar;
+  private translateService: TranslateService;
 
   public rootPage: any;
   public pages: Array<{ title: string, component: any }>;
 
-  constructor(platform: Platform, menu: MenuController, splash: SplashScreen, status: StatusBar) {
+  constructor(platform: Platform, menu: MenuController, splash: SplashScreen, status: StatusBar, translateService: TranslateService) {
 
     this.menu = menu;
     this.platform = platform;
     this.splash = splash;
     this.status = status;
+    this.translateService = translateService;
 
-    this.rootPage = ClickerList;
     this.initializeApp();
-
-    // set our app's pages
-    this.pages = [
-      { title: 'Clickers', component: ClickerList },
-      { title: 'Goodbye Ionic', component: Page2 },
-    ];
   }
 
-  private initializeApp(): void {
-    this.platform.ready().then(() => {
+  private initializeApp(): Promise<any> {
```

Comment:
Can we use the actual return type instead of any? Or is it legitimately returning mutable data?